python/ray/tune/tests/test_multinode_sync.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import time
+import unittest
+
+import ray
+from ray.autoscaler._private.fake_multi_node.test_utils import DockerCluster, \
+    DockerMonitor
+
+
+@ray.remote
+def remote_task(val):
+    return val
+
+
+class MultiNodeSyncTest(unittest.TestCase):
+    def setUp(self):
+        self.cluster = DockerCluster()
+        self.cluster.setup()
+
+        self.monitor = DockerMonitor(self.cluster.config_file)
+        self.monitor.start()
```

Comment:
Should the monitor be part of DockerCluster?

Reply:
Right, that does make sense. I'll update this